app/build.gradle

```diff
@@ -89,17 +155,27 @@ dependencies {
     implementation "org.mozilla.photon:icons:${rootProject.ext.dependencies['mozillaComponents']}"
     implementation "org.mozilla.photon:progress:${rootProject.ext.dependencies['mozillaComponents']}"
 
-    armImplementation "org.mozilla:geckoview-nightly-armeabi-v7a:${rootProject.ext.gecko['version']}"
-    x86Implementation "org.mozilla:geckoview-nightly-x86:${rootProject.ext.gecko['version']}"
-    aarch64Implementation "org.mozilla:geckoview-nightly-arm64-v8a:${rootProject.ext.gecko['version']}"
+    nightlyImplementation "org.mozilla.components:engine-gecko-nightly:${rootProject.ext.dependencies['mozillaComponents']}"
+    nightlyArmImplementation "org.mozilla:geckoview-nightly-armeabi-v7a:${rootProject.ext.geckoNightly['version']}"
+    nightlyX86Implementation "org.mozilla:geckoview-nightly-x86:${rootProject.ext.geckoNightly['version']}"
+    nightlyAarch64Implementation "org.mozilla:geckoview-nightly-arm64-v8a:${rootProject.ext.geckoNightly['version']}"
+
+    betaImplementation "org.mozilla.components:engine-gecko-beta:${rootProject.ext.dependencies['mozillaComponents']}"
+    betaArmImplementation "org.mozilla:geckoview-beta-armeabi-v7a:61.0.20180614135649"
+    betaX86Implementation "org.mozilla:geckoview-beta-x86:61.0.20180614135649"
+    betaAarch64Implementation "org.mozilla:geckoview-beta-arm64-v8a:61.0.20180614135649"
+
+    productionImplementation "org.mozilla.components:engine-gecko:${rootProject.ext.dependencies['mozillaComponents']}"
+    productionArmImplementation "org.mozilla:geckoview-release-armeabi-v7a:60.0.2"
+    productionX86Implementation "org.mozilla:geckoview-release-x86:60.0.2"
+    productionAarch64Implementation "org.mozilla:geckoview-release-arm64-v8a:60.0.2"
```

Comment:
You created constants for the version numbers in the root build.gradle but you are not using them here :)

Reply:
Ah good catch. That means we need to update the sample-browser app as well. I will update and send a PR.

Reply:
Oh, I was afraid that I did that :D